includes/subscriptions/class-wc-payments-subscription-service.php

```diff
@@ -570,6 +571,21 @@ public function prevent_wcpay_subscription_changes( bool $supported, string $fea
 		return in_array( $feature, $this->supports, true ) || isset( $this->feature_support_exceptions[ $subscription->get_id() ][ $feature ] );
 	}
 
+	/**
+	 * Remove pending renewal order creation from admin edit subscriptions page.
+	 *
+	 * @param array $actions Array of available actions.
+	 * @return array Array of updated actions.
+	 */
+	public function prevent_wcpay_manual_renewal( array $actions ) {
+		global $theorder;
+
+		if ( wcs_is_subscription( $theorder ) && self::is_wcpay_subscription( $theorder ) ) {
+			unset( $actions['wcs_create_pending_renewal'] );
+		}
```

Comment:
Perhaps we should [remove this default from `subscriptions-core`](https://github.com/Automattic/woocommerce-subscriptions-core/blob/2a0654bf0385cc3e5b097943aea5589e843b05d0/includes/admin/class-wcs-admin-meta-boxes.php#L173) and re-add it only in subs pro. Thoughts @james-allan ?

IMO `subscriptions-core` should not implement behaviour that is not needed for the WCPay Subscriptions use case. Then we wouldn't need to add hooks to disable pro-only functionality like this.

Reply:
> IMO subscriptions-core should not implement behaviour that is not needed for the WCPay Subscriptions use case.

These actions are needed for creating subscriptions manually and managing manual subscriptions so subscriptions-core will need to maintain them for that case. 

So IMO the approach is to either:

- Keep this PR as it is. Which is fine.
- Or. Add the logic Brian is adding here to core so core holds the context about removing the actions for WCPay Subscriptions. 

Honestly, doing it in WCPay is the cleaner approach IMO because you don't have to juggle core having that logic and then having to override it or add additional context in the WC Subscriptions plugin.